app/views/va_facilities/_index_table_row.html.erb

```diff
@@ -1,10 +1,24 @@
 <% facilities.each do |facility| %>
+  <% path = facility.class.name === "ClinicalResourceHub" ? "/crh/" + facility.visn.number.to_s : va_facility_path(facility)  %>
+  <% if facility.class.name === "ClinicalResourceHub" %>
+    <% created_count = crh_created_practices_count(facility.visn_id) %>
+    <% adopted_count = crh_adopted_practices_count(facility.visn_id) %>
+  <% else %>
+    <% created_count = facility.practices_created_count %>
+    <% adopted_count = facility.practices_adopted_count %>
+  <% end %>
   <tr>
-    <th scope="row" role="rowheader" class="grid-col-4"><a class="usa-link" href='<%= va_facility_path(facility) %>'><%= facility["official_station_name"] %> (<%= facility["common_name"] %>)</a></th>
-    <td class="grid-col-1"><%= facility.street_address_state %></td>
+    <th scope="row" role="rowheader" class="grid-col-4"><a class="usa-link" href='<%= path %>'><%= facility["official_station_name"] %>
+      <% if facility.class.name != "ClinicalResourceHub"  %> (<%= facility["common_name"] %>)<% end %></a>
+    </th>
+    <td class="grid-col-1"><%= facility.class.name === "ClinicalResourceHub" ? facility.visn.state : facility.street_address_state %></td>
```

Comment:
I thought we weren't assigning one state to a CRH

Reply:
Yeah... sorry I was using visn.state field for this... but since they are associated with multiple states... just displaying '-' for CRH state column... also displaying "-" for complexity field... was displaying "N/A"..  lol